<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_custom</class>
 <widget class="QWidget" name="form_custom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>960</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="btn_custom_refresh">
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" rowspan="2">
      <widget class="QTextEdit" name="txt_code"/>
     </item>
     <item row="1" column="0" rowspan="2">
      <widget class="QTreeView" name="tree_filesystem">
       <property name="maximumSize">
        <size>
         <width>256</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="btn_custom_save">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
